src/clients/python/experimental_api_v2/library/__init__.py

```diff
@@ -105,5 +209,126 @@ def is_server_ready(self):
             If unable to get readiness.
 
         """
-        self.response = self.client_stub.get("/api/health/ready")
+        self.response = self._client_stub.get("/api/health/ready")
+        self._last_request_id = _raise_if_error(self.response['NV-Status'])
         return self.response.status_code == 200
+
+    def is_model_ready(self, model_name, model_version):
+        """Contact the inference server and get the readiness of specified model.
+
+         Parameters
+        ----------
+        model_name: str
+            The name of the model
+
+        model_version: int
+            The version of the model
+
+        Returns
+        -------
+        bool
+            True if the model is ready to be served, False if otherwise.
+
+        Raises
+        ------
+        Exception
```

Comment:
InferenceServerException

Reply:
Even though I have modified Exception->InferenceServerException, all other kinds of exceptions can be raised by the underlying python modules. Would it be appropriate to document InferenceServerException here? Should we be catching these exceptions and raising them as an InferenceServerException, at the expense of visibility of the stack?